test_pipeline.py

```diff
@@ -0,0 +1,26 @@
+import unittest, pytz
+from message import NotificationsMessage, PipelineHistory
+from pipeline_manager import Pipeline
+from uuid import UUID
+from datetime import datetime
+
+class TestPipeline(unittest.TestCase):
```

Comment:
Please look through your code and see what is not covered by these test cases?
For example: expired message, verifying log outputs.